File: Lista_Geral_C/question_2.c
// Escreva um programa que leia três números e seus respectivos pesos e calcule a sua  média ponderada.
#include <stdio.h>

int main(){

    float nota1, nota2, nota3, peso1, peso2, peso3, media;


    printf("Informe a primeira nota e seu peso: ");
    scanf("%f%f", &nota1,&peso1);
    printf("Informe a segunda nota e seu peso: ");
    scanf("%f%f", &nota2, &peso2);
    printf("Informe a terceira nota e seu peso: ");
    scanf("%f%f", &nota3, &peso3);

    media = ((nota1 * peso1) + (nota2 * peso2) + (nota3 * peso3)) / (peso1 + peso2 + peso3);

    printf("A media ponderada e: %.1f", media);

    return 0;
}

File: Lista_Geral_Python/comandos de entrada e saida/question_2.py
### Escreva um programa que leia três números e seus respectivos pesos e 
# calcule a sua  média ponderada.

nota_1 = float(input("Informe a primeira nota: "))
peso_1 = int(input("Informe o peso da primeira nota: "))
nota_2 = float(input("Informe a segunda nota: "))
peso_2 = int(input("Informe o peso da segunda nota: "))
nota_3 = float(input("Informe a terceita nota: "))
peso_3 = int(input("Informe o peso da terceira nota: "))

peso_geral = peso_1 + peso_2 + peso_3
media_ponderada = ((nota_1 * peso_1) + (nota_2 * peso_2) + (nota_3 * peso_3)) / peso_geral

print(f"A media ponderada e: {media_ponderada}")